cfgov/unprocessed/js/modules/util/arrays.js

```diff
@@ -0,0 +1,28 @@
+function forEach( array, callback, scope ) {
```

Comment:
Ah, gotcha! Sorry I didn't look at the implementation closely. Naming it `forEachNode` or something like that may be clearer.

Also, a classic loop may be a cleaner alternative in many cases. For example, the following bit as a classic loop doesn't necessitate creating a function:

```
arrayHelpers.forEach( _list.children, function( i ) {
    _list.children[i].classList.remove( 'filter-match' );
} );
```

As classic loop:
```
for ( var i, len = _list.children.length; i < len; i++ ) {
    _list.children[i].classList.remove( 'filter-match' );
}
```